.travis.yml

```diff
@@ -1,19 +1,35 @@
 os:
     - linux
-    - windows
```

Comment:
right now this will quietly fail, but i have plans to use this for .exe build...

Reply:
As I said, you're welcome to try re-enabling them, I'm just not sure it'll even make it through the build process.

Reply:
That is pretty much, why i had it in `allow_failures`... Just so we can test it, but don't have to worry about it.

Reply:
I've spent all of the time I can on this for now, but I've marked this PR as editable by contributors (via the poorly-named `travis_testing_squashed` branch).  I'm totally OK with you making whatever modifications you feel like you need to make if you'd like to merge this; I'm sure many people would appreciate our having at least a little bit of test coverage.